.env.example

```
@@ -30,3 +30,5 @@ ZOOM_ACCOUNT_ID=                # Zoom account ID
 ZOOM_CLIENT_ID=                 # OAuth app client ID 
 ZOOM_CLIENT_SECRET=             # OAuth app client secret (keep secure!)
 ZOOM_WEBHOOK_SECRET_TOKEN=      # Oauth app secret token (for webhook events)
+# For testing only - bypasses webhook signature validation
+MOCK_ZOOM_WEBHOOK=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MOCK_ZOOM_WEBHOOK key should go before the ZOOM_ACCOUNT_ID key

(UnorderedKey)